sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -259,10 +244,12 @@ def build_image_from_func(self, component_func, namespace, base_image, timeout):
 
```

Comment:
`build_image_from_func`'s code look almost the same as `build_image_from_dockerfile`. Can you make `build_image_from_func` use `build_image_from_dockerfile`?

Reply:
You are absolutely right. It might require a little effort to update the prepare_docker_tarball_with_py in DockerfileHelper. How about I'm addressing this in another PR since this PR aims to enable the unit tests.
Thanks for the suggestion of refactoring.

Reply:
>How about I'm addressing this in another PR since this PR aims to enable the unit tests.

Ok.